apps/aecore/src/aec_governance.erl

```diff
@@ -91,9 +93,14 @@ minimum_gas_price() ->
     0.
 
 %% In key blocks / generations
+%% NOTE: The delay must be at least two generations, otherwise the
+%% payout would be done before anyone had the chance to report the
+%% miner for fraudulent behavior.
 beneficiary_reward_delay() ->
-    aeu_env:user_config_or_env([<<"mining">>, <<"beneficiary_reward_delay">>],
-                               aecore, beneficiary_reward_delay, ?BENEFICIARY_REWARD_DELAY).
+    Delay = aeu_env:user_config_or_env([<<"mining">>, <<"beneficiary_reward_delay">>],
```

Comment:
I know not introduced of this PR so please consider as optional, though `beneficiary_reward_delay` shall be in `/apps/aeutils/test/data/epoch_full.yaml`.